docarray/document/mixins/video.py

```diff
@@ -110,16 +110,36 @@ def load_uri_to_video_tensor(
         import av
 
         with av.open(self.uri, **kwargs) as container:
-            if only_keyframes:
-                stream = container.streams.video[0]
-                stream.codec_context.skip_frame = 'NONKEY'
+            stream = container.streams.video[0]
+            stream.codec_context.skip_frame = 'NONKEY'
 
-            frames = []
+            key_frames = []
             for frame in container.decode(video=0):
+                img = frame.to_image()
+                key_frames.append(img)
+
+        with av.open(self.uri, **kwargs) as container:
+            i = 0
+            curr_key_frame = key_frames[i]
+            key_frame_indices = []
+            frames = []
+            for j, frame in enumerate(container.decode(video=0)):
                 img = frame.to_image()
                 frames.append(np.asarray(img))
 
-        self.tensor = np.moveaxis(np.stack(frames), 1, 2)
+                if (np.asarray(img) == np.asarray(curr_key_frame)).all():
```

Comment:
I am not a big fan of doing this. 
From what I understand av is able to tell when it load the video which frame is a key frame. It means that the info is either store in the file or that av can determine it. In both case we should do smth like this.

```python
for j, frame in enumerate(container.decode(video=0)):
    frames.append(np.asarray(img))
    if is_key_frame(frame):
        key_frame_indices.append(j)

```
the question is how de we do the `is_key_frame` but I believe av must have it